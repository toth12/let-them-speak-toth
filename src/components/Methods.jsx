import React from 'react';
import Hero from './Hero';

const Methods = props => (
  <div className='methods'>
    <Hero text='Methodology' />
    <div className='container'>
      <h2>Subheading here for page section</h2>
      <p>Lorem ipsum dolor sit amet, arcu vulputate egestas dolor wisi, fugiat vestibulum etiam mattis sit, curabitur elit nulla vel, mus quis elit porttitor nulla. Amet elit in convallis, metus nam at sed, ipsum phasellus a dapibus, ornare in massa pharetra. Montes elementum pede integer ac, pulvinar aliquam non in augue, orci nulla eros nulla. Vulputate ut sed eget eu, dolor aliquam velit quisque a. Fermentum venenatis rhoncus vitae congue, suspendisse potenti pellentesque vestibulum.</p>
      <p>Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse. A sit, vehicula eleifend, per massa vitae. Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse.</p>
      <h2>Subheading here for page section</h2>
      <table className='image-trio'>
        <tbody>
          <tr>
            <td>
              <a href='/assets/files/sample.pdf' download>
                <div className='background-image' />
                <div>Essay title number one<br/>Subtitle for this essay</div>
              </a>
            </td>
            <td>
              <a href='/assets/files/sample.pdf' download>
                <div className='background-image' />
                <div>Essay title number two<br/>Subtitle for this essay</div>
              </a>
            </td>
            <td>
              <a href='/assets/files/sample.pdf' download>
                <div className='background-image' />
                <div>Essay title number three<br/>Subtitle for this essay</div>
              </a>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
)

export default Methods;