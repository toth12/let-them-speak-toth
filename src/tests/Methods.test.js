import React from 'react';
import Methods from '../components/Methods';
import { createBrowserHistory } from 'history'
import { Router } from 'react-router';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer
    .create(
      <Router history={createBrowserHistory()}>
        <Methods/>
      </Router>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});