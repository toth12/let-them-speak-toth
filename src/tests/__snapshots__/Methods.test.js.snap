// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="methods"
>
  <div
    className="hero background-image"
  >
    <h1>
      Methodology
    </h1>
  </div>
  <div
    className="container"
  >
    <h2>
      Subheading here for page section
    </h2>
    <p>
      Lorem ipsum dolor sit amet, arcu vulputate egestas dolor wisi, fugiat vestibulum etiam mattis sit, curabitur elit nulla vel, mus quis elit porttitor nulla. Amet elit in convallis, metus nam at sed, ipsum phasellus a dapibus, ornare in massa pharetra. Montes elementum pede integer ac, pulvinar aliquam non in augue, orci nulla eros nulla. Vulputate ut sed eget eu, dolor aliquam velit quisque a. Fermentum venenatis rhoncus vitae congue, suspendisse potenti pellentesque vestibulum.
    </p>
    <p>
      Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse. A sit, vehicula eleifend, per massa vitae. Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse.
    </p>
    <h2>
      Subheading here for page section
    </h2>
    <table
      className="image-trio"
    >
      <tbody>
        <tr>
          <td>
            <a
              download={true}
              href="/assets/files/sample.pdf"
            >
              <div
                className="background-image"
              />
              <div>
                Essay title number one
                <br />
                Subtitle for this essay
              </div>
            </a>
          </td>
          <td>
            <a
              download={true}
              href="/assets/files/sample.pdf"
            >
              <div
                className="background-image"
              />
              <div>
                Essay title number two
                <br />
                Subtitle for this essay
              </div>
            </a>
          </td>
          <td>
            <a
              download={true}
              href="/assets/files/sample.pdf"
            >
              <div
                className="background-image"
              />
              <div>
                Essay title number three
                <br />
                Subtitle for this essay
              </div>
            </a>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
